modules/installation-gcp-permissions.adoc

```diff
@@ -18,6 +19,12 @@ account requires the following permissions:
 * Service Account User
 * Storage Admin
 
+ifeval::["{context}" == "installing-gcp-user-infra"]
+.Additional required roles for GCP UPI installations
```

Comment:
Not "UPI" anywhere ever. Maybe "Required roles for user-provisioned GCP infrastructure"